system-services/usb-rauc-install.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Meticulous manual updater
+Requires=rauc.service
+
+[Service]
+Type=simple

```

Comment:
This gets started by udev once, right? I think there is a type oneshot for that

Reply:
Yes, just checked that out, it is documented for `simple` and `oneshot` to have similar behavior, excepting the time when the service is reported to be up, the first one considered up, as the service starts, and the latter until the binary execution ends. Though I agree that `oneshot` is also more descriptive

Reply:
I found something, while addressing this, as the `oneshot` service is considered up until its executable ends, calling the `systemctl start usb-rauc-install.service` blocks the execution of the shell script its being called in, so its better to leave it as `simple`